P0009/P0009.bs

```diff
@@ -1035,27 +1024,18 @@ template<ptrdiff_t... LHS, ptrdiff_t... RHS>
 
 1. A *layout mapping policy* is a class that contains a *layout mapping*, a nested class template.
 
-2. A *layout mapping policy* meets the requirements in Table �.
+2. A *layout mapping policy* and its *layout mapping* nested class template meet the requirements in Table �.
 
 3. A *layout mapping* meets the requirements of *Cpp17DefaultConstructible*, *Cpp17CopyAssignable*, *Cpp17EqualityComparable*, and the requirements in Table �.
 
 4. In Table �:
     * `MP` denotes a layout mapping policy.
+    * `M` denotes a specialization of the layout mapping policy's nested layout mapping template class.
     * `E` denotes a specialization of `extents`.
-    * `e` denotes an object of type `E` defining a multidimensional index space.
-    * `r` is a value of an integral type such that 0 <= `r` < `e.rank()`.
-    * If `k...` is a pack of an integer type, we say that "`k...` is in `e`" 
-      if `k...` denotes a multi-index in `e`.
-      Let the packs of integer type `i...` and `j...` be in `e`.
-      Denote the <math>r</math>-th member of packs `i...` and `j...` by 
-      <math>i<sub>r</sub></math> and <math>j<sub>r</sub></math>, respectively. 
-      Let `r` equal <math>r</math>.
-      `sizeof...(i)==E::rank()` is `true`, 
-      0 <= <math>i<sub>r</sub></math> < `e.extent(r)`, 
-      `sizeof...(j)==E::rank()`, and
-      0 <= <math>j<sub>r</sub></math> < `e.extent(r)`.
-    * `M` denotes a layout mapping class.
-    * `m` denotes an object of type `M` that maps a pack `i...` denoting a multi-index to an integral value.
+    * `e` denotes an extents object of type `E`
+    * `m` denotes a layout mapping object of type `M`.
```

Comment:
"`m` denotes an object of type `M`."  (We already know that it's a layout mapping object; see above.)